File: magma-platform/magma-platform-bungee/src/main/java/de/jalumu/magma/platform/bungee/text/placeholder/BungeePlaceholderProvider.java
package de.jalumu.magma.platform.bungee.text.placeholder;

import de.jalumu.magma.text.placeholder.PlaceholderProvider;
import net.kyori.adventure.text.minimessage.tag.resolver.TagResolver;

import java.util.UUID;

public class BungeePlaceholderProvider extends PlaceholderProvider {
    @Override
    protected TagResolver getGlobal() {
        return TagResolver.builder().build();
    }

    @Override
    protected TagResolver getPlayer(UUID uuid) {
        return TagResolver.builder().build();
    }
}

File: magma-platform/magma-platform-bungee/src/main/kotlin/de/jalumu/magma/platform/bungee/text/placeholder/BungeePlaceholderProvider.kt
package de.jalumu.magma.platform.bungee.text.placeholder

import de.jalumu.magma.text.placeholder.PlaceholderProvider
import net.kyori.adventure.text.minimessage.tag.resolver.TagResolver
import java.util.*

class BungeePlaceholderProvider : PlaceholderProvider() {
    override fun getGlobal(): TagResolver {
        return TagResolver.builder().build()
    }

    override fun getPlayer(uuid: UUID): TagResolver {
        return TagResolver.builder().build()
    }
}
